packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -60,6 +61,22 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     log('THIS address: ', this.outcomeAddress);
   }
 
+  async testTorrentingCapability(timeOut: number) {
+    log('Testing torrenting capability...');
+    const gotAWire = new Promise(resolve => {
+      super.add(mockTorrents[0].magnetURI, (torrent: Torrent) => {
+        torrent.once('wire', wire => {
+          this.remove(torrent.infoHash);
+          resolve(true);
+        });
+      });
+    });
+    const timer = new Promise(function(resolve, reject) {
+      setTimeout(resolve, timeOut);
+    });
+    return Promise.race([gotAWire, timer]);
```

Comment:
If you are on a slow computer / connection, you might see the warning unnecessarily. 

Increasing the timeout would fix that, but you don't see the warning until the timeout lapses.